### PR TITLE
Add (real) support for IDEA build 201

### PR DESCRIPTION
**Pull request by fabmars**
Following https://github.com/OpherV/gitflow4idea/pull/288
Considering 2020 deprecations
image
and based on https://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/plugin_components.html

Here is the support for the gitflow plugin on IDEA 201.

### CODE REVIEW

**Comment by fabmars** [comment]
Should solve #291 #292 #293

**Comment by fabmars** [comment]
and #276

**Comment by OpherV** [comment]
Brilliant! Thanks for the effort - I'll try and review over the weekend and hopefully get a version out for early next week 😄

**Comment by fabmars** [comment]
The handleMerge() logic is really hacky but that's an issue with idea's apis, nothing we can do. I tried to improve it though. That's what took 80% of the time spent here.

File: src/main/java/gitflow/ui/GitflowTaskDialogPanelProvider.java [hunk 1]
```diff
@@ -13,14 +12,19 @@
...
+    @Deprecated
     @Nullable
     @Override
     public TaskDialogPanel getOpenTaskPanel(@NotNull Project project, @NotNull Task task) {
+        return null;
+    }
+
+    @Nullable
+    @Override
+    public TaskDialogPanel getOpenTaskPanel(@NotNull Project project, @NotNull LocalTask task) {
```
Comment (OpherV) [inline src/main/java/gitflow/ui/GitflowTaskDialogPanelProvider.java:27]: What's this change about?

Comment (OpherV) [reply src/main/java/gitflow/ui/GitflowTaskDialogPanelProvider.java:27]: Oh it's one of the deprecations. Nevermind :)

**Comment by OpherV** [comment]
Getting this while testing this PR:

image

**Comment by OpherV** [comment]
Solved by refactoring some of `GitflowVerisonTester` and its init functionality

**Pull request merged**